src/runtime/admin/component-manager.spec.ts

```diff
@@ -1,9 +1,174 @@
-import { describe, test } from 'vitest'
+import { describe, expect, test, vi } from 'vitest'
+import * as vue from 'vue'
+import { nextTick, reactive } from 'vue'
+import ComponentManager from '#cwa/runtime/admin/component-manager'
 
-describe('Manager class tests', () => {
-  describe('addToStack function', () => {
-    test.todo('If the last click target does not equal the new click target, clear the stack and save the new click target', () => {})
+function createComponentManager (mockStore?: any) {
+  const mockAdminStore = {
+    useStore: () => mockStore || ({
+      state: {
+        isEditing: false
+      }
+    })
+  }
 
-    test.todo('Adds a stack object to the stack array', () => {})
+  const manager = new ComponentManager(mockAdminStore)
+
+  return {
+    manager,
+    store: mockAdminStore
+  }
+}
+
+describe('Component Manager', () => {
+  describe('store getter', () => {
```

Comment:
describes should be the exact function name and function e.g. adminStore getter function 
or
resetStack function

And then descriptors of the desired functionality in nested description or test functions.

It makes reading tests and finding the relevant tests much easier. Thanks

Reply:
fixed